.github/workflows/test-go.yaml

```diff
@@ -73,6 +73,7 @@ jobs:
           NETWORK_TEST=1 \
           REDIS_TEST=1 \
           MYSQL_TEST=1 \
+          FILE_STORAGE_TEST=1 \
```

Comment:
Should this be `MINIO_STORAGE_TEST=1`? (It actually requires a Minio server running, that happens to be using file as backend)

Reply:
it happens to be `minio` but I wanted to keep our options open. Happy to make it more specific

Reply:
This is akin to current `MYSQL_TEST` config, where the SQL backend it happens to be `mysql` :)

Reply:
sgtm! plus the `minio` name is cool :D Pushed a fix.